File: Difficulty: Easy/Print Square Wall Using Nested Loops/print-square-wall-using-nested-loops.py
#{ 
 # Driver Code Starts
#Initial Template for Python 3

# } Driver Code Ends

def print_square_wall(n):
    # Outer loop for rows
    for i in range(n):
        # Inner loop for columns in each row
        for j in range(n):
            # Print '*' without a newline at the end
            print('*', end=' ')
        # Print a new line after each row
        print()

# Example usage:
n = int(input())  # Get the size of the square wall
print_square_wall(n)

#{ 
 # Driver Code Starts.

# } Driver Code Ends

File: Difficulty: Easy/Print Square Wall Using Nested Loops/print-square-wall-using-nested-loops.java
//{ Driver Code Starts
// Initial Template for Java

import java.util.Scanner;

class Main {

    
// } Driver Code Ends

// User function Template for Java

public static void squareWall(int s) {
    // Iterate through each row of the square wall
    int i = 1;
    while (i <= s) {
        // For each row, iterate through each column
        int j = 1;
        while (j <= s) {
            // Print '*' with a space for each column
            System.out.print("* ");
            j++;
        }
        // Move to the next line after completing one row
        System.out.println();
        i++;
    }
}


//{ Driver Code Starts.

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int t = scn.nextInt();
        while (t-- > 0) {
            int s = scn.nextInt();
            squareWall(s);

            System.out.println("~");
        }
        scn.close();
    }
}
// } Driver Code Ends